api/resources/translations/messages-es.properties

```
@@ -229,15 +229,15 @@ Set\ as\ default\ application\ language = Seleccionar como idioma por defecto de
 Set\ as\ language\ for\ outgoing\ messages = Seleccionar como idioma para mensajes salientes
 Settings = Ajustes
 Showing\ number\ of\ total = Mostrando {{number}} de {{total}}.
-Skipped\ number\ of\ records = Se saltaron {{number}} registros.
+Skipped\ number\ of\ records = Se omitieron {{number}} registros.
 Start = Iniciar
 Start\ messages\ based\ on = Iniciar mensajes en función de
 Start\ new\ conversation = Empezar nueva conversación
 Submit = Enviar
 Submit\ Report = Enviar informe
-Submit\ via\ either = 
-Submit\ via\ sms = 
-Submit\ via\ web = 
+Submit\ via\ either = Enviar vía
```

```suggestion
Submit\ via\ either = Trate primero con Internet y luego con SMS
```

Like before, I couldn't find the usage, but based on the English `Try internet first, then SMS`, I've suggested something along the same lines as the message intention.